tests/general_indicator_tests/general_indicator_test_helpers.py

```diff
@@ -3,9 +3,14 @@
 accessors = Accessors()
 
 
-def get_by_geo():
+def get_by_geo(housing_security=False, pums_demographics=False):
     """housing security parameter is temporary"""
-    accessors_list = accessors.all
+    if housing_security:
```

Comment:
I think that in `dev` get_by_geo shouldn't have any parameters. Tests on particular categories for developments should stay on local branches and not be merged in